Automattic/wp-calypso
# add search free to products list

## Proposed Changes

* This product adds the search free product to the Jetpack products array.
* This is to fix some bugs where other code attempts to look up the `jetpack_search_free` slug in the products array

## Testing Instructions

* Obtain a Jetpack test site
* Be sure to start disconnected from Jetpack
* Navigate to https://wordpress.com/checkout/jetpack/jetpack_search_free?connect_after_checkout=true&admin_url=https%3A%2F%2FYOUR_SITE_URL%2Fwp-admin%2F&from_site_slug=YOUR_SITE_URL&source=my-jetpack&redirect_to=https%3A%2F%2FYOUR_SITE_URL%2Fwp-admin%2Fadmin.php%3Fpage%3Djetpack-search&unlinked=1 - Replace `YOUR_SITE_URL` with the URL of your site (excluding https://)
* The purchase should fail and you should land on an error screen
![Screenshot 2024-02-27 at 2 05 55 PM](https://github.com/Automattic/wp-calypso/assets/18016357/30f7440d-febc-46d8-b421-40cd8cf4697c)

* Make sure that you are still disconnected on the test site and have no search free product
* Now, start up Calypso with this PR applied
* Using the same checkout URL, except with the `calypso.localhost` host, try the checkout again
* This time, the checkout should complete successfully and you should be asked to connect Jetpack after the purchase

## Pre-merge Checklist
